portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -57,4 +79,56 @@ private String convertToJson(ArrayList <String>benjaminList){
     json += "}";
      return json;
   }
+
+
+   @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    long timestamp = System.currentTimeMillis();
+    // Get the input from the form.
+    String text = getParameter(request, "text-input", "");
+    boolean upperCase = Boolean.parseBoolean
```

Comment:
Nit: leave the ( on the same line here. It's a common style practice if you put the parameters in the next line. i.e. `Boolean.parseBoolean(`

Same below